### ISSUE TITLE
The scene dialog displays with cells around every line

### ISSUE THREAD

**Issue by moorepants** [pydy/pydy#244]
![selection_100](https://cloud.githubusercontent.com/assets/276007/8238994/f0b64a28-15ad-11e5-87f0-5a18235d679d.png)

Maybe we should also remove this dialog thing for 0.3.0.

**Comment by sahilshekhawat** [comment]
Yes, We have already removed loading simulation files and probably we should remove showing json files too.

**Comment by moorepants** [comment]
This is the scene file not the simulation file. See the screen shot.

**Comment by moorepants** [comment]
Yes.

**Comment by oliverlee** [comment]
Do we want to fix the dialog thing since we've passed 0.3.0?

**Comment by moorepants** [comment]
We should still fix it if it is still a bug. We can either have a bugfix release or just wait till next full release.
